src/patternfly/components/Tabs/tabs.scss

```diff
@@ -304,152 +304,156 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
   }
 
   // Vertical
-  &.pf-m-vertical {
-    --pf-c-tabs--Width: var(--pf-c-tabs--m-vertical--Width);
-    --pf-c-tabs--inset: var(--pf-c-tabs--m-vertical--inset);
-    --pf-c-tabs--before--BorderBottomWidth: 0;
-    --pf-c-tabs__link--PaddingTop: var(--pf-c-tabs--m-vertical__link--PaddingTop);
-    --pf-c-tabs__link--PaddingBottom: var(--pf-c-tabs--m-vertical__link--PaddingBottom);
-    --pf-c-tabs__link--before--Left: 0;
-    --pf-c-tabs__link--disabled--before--BorderBottomWidth: 0;
-    --pf-c-tabs__link--disabled--before--BorderLeftWidth: var(--pf-c-tabs--before--border-width--base);
-    --pf-c-tabs__link--after--Top: 0;
-    --pf-c-tabs__link--after--Bottom: 0;
-    --pf-c-tabs__link--after--Right: auto;
-    --pf-c-tabs__list--ScrollSnapTypeAxis: var(--pf-c-tabs--m-vertical__list--ScrollSnapTypeAxis);
-
-    display: inline-flex;
-    flex-direction: column;
-    height: 100%; // If not a flex child, set height
-    padding: 0; // Because vertical variant has no scroll buttons, reset padding
-    overflow: visible; // remove in breaking change release
-
-    .pf-c-tabs__list {
-      position: relative;
+  @media screen and (min-width: $pf-global--breakpoint--md) {
```

Comment:
I feel like it's hard to answer this, in general. I think that answer depends on how many tabs you have. The examples in the demos work well because there are only two tabs. If there were more tabs, then changing to the expandable menu might be a better solution.

Reply:
How do expandable vertical tabs work? When opened, does it push down the content, does it overlay it, or does it push it to the right? I'm assuming it pushes down the content when expanded, and clicking on a tab would change the content underneath, and you'd have to collapse the tabs or scroll down to see the content for that tab right?

If that's the case then yeah, I don't think that's too bad of an experience! It's almost like a navigation that you collapse and expand as needed.

Reply:
> I'm assuming it pushes down the content when expanded, and clicking on a tab would change the content underneath, and you'd have to collapse the tabs or scroll down to see the content for that tab right?

@mmenestr yep! https://www.patternfly.org/v4/components/tabs#vertical-expandable

Reply:
@mattnolting looks like expandable tabs also rely on `.pf-m-vertical`, so wrapping the media query around `.pf-m-vertical` breaks the ability for expandable to work.

I think we should group `.pf-m-vertical` and `.pf-m-expandable` for the base vertical styling, like we do in jump links. wdyt?

https://github.com/patternfly/patternfly/blob/dd3439944fa37e1bb97131c4a81b277a8eb78ed1/src/patternfly/components/JumpLinks/jump-links.scss#L89-L90